docs/reference/tags/package_step_product_search_tag/index.md

```diff
@@ -64,10 +64,35 @@ The tag returns an `items` array and a `Paginate` pagination object to the block
 
 ## Search Params
 
-#### include_organisation_products
+#### `include_organisation_products`
 
 This allows you to find the step products if the current package step is associated with a company that is different to the company for the current site, but they are linked by the same organisation. e.g. `include_organisation_products: true`
 
+## Tag-based filtering
+
+If the company has added tags to the step's products, these can be used to filter the results outputted by the search tag.
+
+Product tags are textual values that identify attributes of the products. They are grouped under categories which the company is also able to define.
+
+Some examples of these might be "Star Rating", "Amenities", "Distance" etc. Consult the company for the available tags and categories.
+
+Filtering is aware of `include_organisation_products` and will include the organisation's products if the flag is set.
+
+**Syntax**
+
+`package_step_product_search` filters results if one or more `search` query parameters are present in the URL. The search query parameters are in the format `search[<tag category>][]=<tag value>`; add as many as needed to filter the results.
+
+Multiple categories and values by repeating the `search[][]` query parameter.
```

Comment:
**nit** This feels a bit cryptic. Should we say something like: "You can specify multiple categories and values by repeating the `search[][]` query parameter.`?